testdriver/testdriver.go

```diff
@@ -887,6 +887,17 @@ func Condition(cond string) (bool, error) {
 	if !semver.IsValid(v) {
 		return false, fmt.Errorf("%v is not a valid semver version", v)
 	}
+	// We have version information. Be perhaps a bit overly
```

Comment:
Nice change. Maybe we can phrase it a bit different to differentiate `vim` and `vim:v8.2.x`? I think that a future me might be confused when reading tests.

I don't have a nice suggestion on how to improve though. `vimver:v8.2.xxxx` for version?
Or `flavor:vim` / `flavor:gvim`?

Something that differentiate version `vim` from flavour `vim` would be nice.

Reply:
Thanks. It wasn't sitting quite right with me either. So I simplified things even further in https://github.com/govim/govim/pull/1098. Please let me know what you think. The tests hopefully make clear what's going on now too.